src/middlewared/middlewared/api/v25_04_0/docker.py

```diff
@@ -25,6 +35,16 @@ class DockerEntry(BaseModel):
     pool: NonEmptyString | None
     nvidia: bool
     address_pools: list[AddressPool]
+    cidr_v6: IPvAnyInterface
```

Comment:
Why would this be of type `IPvAnyInterface` if we just accept an ipv6 address?

Reply:
@yocalebo validation for this has been added below - we don't have a type from pydantic which highlights ipv4/ipv6 interfaces separately or do you think we should perhaps use `ipaddress` directly in this case?

Reply:
We don't want to use pydantic field types if we don't have to. Especially if there is a type that is builtin to python. So yeah, specifing `ipaddress.IPv6*` type is preferred here.